packages/langium/test/grammar/type-system/inferred-types.spec.ts

```diff
@@ -595,13 +595,18 @@ function getType(types: AstTypes, name: string): InterfaceType | UnionType | und
     return interfaceType || unionType;
 }
```

Comment:
I think the file should be named with the suffix .test.ts

Reply:
I have no issue with that 👍 , it is the odd one out in this folder.